CONTRIBUTING.md

```diff
@@ -187,6 +187,39 @@ Tool-specific-repository.org:
   * recommended: list these FMUs and/or provide links to the repository of the exporting tool
   * optionally additional information needed to reproduce the results of the importers (e.g., compiler flags)
 
+#### CSV Rules for inout  and result files for compatibility information
+
+CSV as input, output and reference file format is convenient as many tools support CSV import and export.
+However, CSV is not restrictive enough to allow easy exchange of time-based signals.
+Here we will declare a number of additional restrictions on top of the CSV format to ease handling:
+
+- Separator: comma (`,`): separators may only be used between elements, not the end of a line
+
+- Numbers must be unquoted and specified in the format used for floating point literals as in the C programming language (without the type suffix).
+
+- All numeric cell entries contain numbers, labels for enumerations are not allowed (it would require readers to have access to the FMU information).
+Boolean values should be expressed as `0` and `1`.
+
+- The first column contains the time.
+The values in time must be monotonically increasing.
+
+- The first line contains the variable names quoted with double quotes (`"`)
+
+- All variable names are exactly the same as defined in the modelDescription.xml (no additional variables are allowed)
+
+- Starting with line 2, data is supplied (no units in line 2, no comments allowed)
+
+- A CSV file must not be larger than 1 Megabyte.
+
+**Example:**
```

Comment:
@chrbertsch  It would be nice to have an example of how input array is supposed to be specified. What if "input" is a 2-element vector instead?

Reply:
Similarly to how I understand your proposal in https://github.com/CATIA-Systems/FMPy/issues/495, we could define, that for array inputs, one uses the serialization of array variables as difined in the FMI 3.0 spec 2.2.7.1 https://fmi-standard.org/docs/3.0/#fmu-variables, and repeats the array variable name in line 1 as often as there are elements in the array, e.g.  "time","input_A","input_A", "input_A", "input_A","input_B" if input_A is .e.g a 2x2 arrray.
Or is there a different proposal , how to handle this?
@klausschuch : any idea from your side?

Reply:
More generally the question is, if we could in the input.csv file e.g. also support binary or string data, or whether we restrict  it to simple use cases.
@pmai : What do you think?

Reply:
FMI design meeting
Pierre Use the same spec for binary as for the start values in FMI 3.0
For arrays we  could use the strucutured naming convension with brackets A[1][1]. 
We should define if zero or one-based. 
Andreas: would prefer zero based. 
Pierre: we could point to the clarification for the ports and icons array treatment.